packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -2460,6 +2460,48 @@ void main() {
     expect(node.value, 'Item 3');
   });
 
+
+  testWidgets('Semantics does not include initial menu buttons', (WidgetTester tester) async {
+    final ThemeData themeData = ThemeData();
+    final TextEditingController controller = TextEditingController();
+    addTearDown(controller.dispose);
+
+    await tester.pumpWidget(
+      StatefulBuilder(
```

Comment:
nit: probably we don't need StatefulBuilder as well. Just wanted to simplify this test:)